frontend/src/Pages/FeedPage.tsx

```diff
@@ -53,12 +57,26 @@ export default function FeedPage() {
         document.title = docTitle;
     }, [siteInfo, feedType]);
 
+    const handleFeedSortingChange = async (e: React.MouseEvent) => {
+        e.preventDefault();
+        const feedSorting = (e.currentTarget as HTMLAnchorElement).dataset.feedSorting;
```

Comment:
Is there a reason to read DOM attributes here? Seems like an easier way would be to pass the sorting value as a parameter to the function:

```
const handleFeedSortingChange = (value: FeedSorting) => async (e: React.MouseEvent) => {
 ...
}

<a href='#' className={sorting === FeedSorting.postCommentedAt ? styles.active : ''} onClick={handleFeedSortingChange(FeedSorting.postCommentedAt)}>LIVE</a>
```